Description: FTBFS on armel, armhf due to not defined GL_MULTISAMPLE
Author: Tobias Frost <tobi@debian.org>
Bug: https://github.com/FreeCAD/FreeCAD/issues/18707
Forwarded: not-yet, awaiting feedback from upstream.
Applied-Upstream: <version|URL|commit, identifies patches merged upstream, optional>
Reviewed-by: <name and email of a reviewer, optional>
Last-Update: 2024-12-24 <YYYY-MM-DD, last update of the meta-information, optional>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/Mod/CAM/PathSimulator/AppGL/DlgCAMSimulator.cpp
+++ b/src/Mod/CAM/PathSimulator/AppGL/DlgCAMSimulator.cpp
@@ -225,6 +225,13 @@
     setAnimating(true);
 }
 
+// tobi@d.o: This is to fix compilation error on armhf, armel
+// however, GL_MULTISAMPLE should be by default on, see
+// https://registry.khronos.org/OpenGL-Refpages/gl4/html/glEnable.xhtml
+#ifndef GL_MULTISAMPLE
+#define GL_MULTISAMPLE  0x809D
+#endif
+
 void DlgCAMSimulator::initialize()
 {
     mMillSimulator->InitSimulation(mQuality);
